src/schemas/json/skypilot-task.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/skypilot-task.json",
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "workdir": {
+      "type": "string"
+    },
+    "event_callback": {
+      "type": "string"
+    },
+    "num_nodes": {
+      "type": "integer"
+    },
+    "resources": {
+      "type": "object"
+    },
+    "file_mounts": {
+      "type": "object"
+    },
```

Comment:
Can we add a detailed schema for [storage style `file_mounts`](https://skypilot.readthedocs.io/en/latest/reference/yaml-spec.html) here?

```
/datasets-storage:
    name: sky-dataset  # Name of storage, optional when source is bucket URI
    source: /local/path/datasets  # Source path, can be local or s3/gcs URL. Optional, do not specify to create an empty bucket.
    store: s3  # Could be either 's3' or 'gcs'; default: None. Optional.
    persistent: True  # Defaults to True; can be set to false. Optional.
    mode: MOUNT  # Either MOUNT or COPY. Optional.
```